### PR TITLE
fix: add corresponding message for override error in js files

### LINKED ISSUE #45591: noImplicitOverride reports in JS files with checkJs enabled, can only ts-ignore the line
# Bug Report

### 🔎 Search Terms

override checkjs jsdoc

### 🕗 Version & Regression Information

- This changed between versions 4.2 and 4.3

### ⏯ Playground Link

[Playground Link](https://www.typescriptlang.org/play?noImplicitOverride=true&filetype=js#code/MYGwhgzhAEBiD29oG8CwAoa0C2BTALgBbwAmAFAJQoZZYBOBArnQHbQCMATAMwAsN0AL4Zh6DKEgwAQmDrRcAD3y4WJGAiRpMOAsXJUttaA3zM2ERgAdcdAHR4ipStADU0XpwDcA0YKA)

### 💻 Code

```tsx
class Foo {
  method() {
    return 1234
  }
}

class Bar extends Foo {
  method() {
    return super.method() + 42;
  }
}
```

### 🙁 Actual behavior

In my projects, I set `checkJs` to enforce that our non-TS files also typecheck. We also use the `noImplicitOverride` rule.

But, if you try to create a class with method overrides in JS files, you get the override errors, even though there doesn't appear to be a way to silence them except via `// @ts-ignore`.

### 🙂 Expected behavior

`noImplicitOverride` is not reported in JS files (or, there's some JSDoc way to add it, like types can be).

### PR DESCRIPTION
**Pull request by stkevintan**
Fixes #45591

### CODE REVIEW

File: src/compiler/diagnosticMessages.json [hunk 1]
```diff
@@ -3817,6 +3817,14 @@
         "category": "Error",
         "code": 4118
     },
+    "This member must have a leading JSDoc comment with an '@override' tag because it overrides a member in the base class '{0}'.": {
```
Comment (sandersn) [inline src/compiler/diagnosticMessages.json:3820]: I would remove 'leading' in the errors. It doesn't add anything, either for beginners or advanced Typescripters.

**Review by sandersn — CHANGES_REQUESTED** [review]
Looks good, I would just like the word 'leading' removed.

File: src/compiler/diagnosticMessages.json (hunk 1 above)
Comment (stkevintan) [reply src/compiler/diagnosticMessages.json]: Sure, updated

**Comment by sandersn** [comment]
@stkevintan Looks like there are 3 lint errors still. Once you fix those I will merge this.

**Pull request merged**